R/plot_biomass.R

```diff
@@ -0,0 +1,121 @@
+#' Plot Total Biomass
+#'
+#' @template dat
+#' @template model
```

Comment:
The use of `#' @template` is outdated. Guidelines suggest that package developers use `#' @inheritParams`. See guidance here on "[Reusing documentation](https://cran.r-project.org/web/packages/roxygen2/vignettes/reuse.html)".

Reply:
Thanks for letting us know kelli! I have made the change following the guidance you have provided :)